ablySpec/TestUtilities.swift

```diff
@@ -327,6 +327,8 @@ class MockHTTPExecutor: NSObject, ARTHTTPExecutor {
 class MockTransport: ARTWebSocketTransport {
 
     var lastUrl: NSURL?
+    var lastSentMessage: ARTProtocolMessage?
+    var connectedMessage: ARTProtocolMessage?
```

Comment:
BTW. MockTransport is not a great name given it's not really a mock, but instead a Proxy.